app/frontend/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,17 @@
+import ErrorFallback from "components/ErrorFallback";
+import { PropsWithChildren, useCallback } from "react";
+import { ErrorBoundary as ReactErrorBoundary } from "react-error-boundary";
+
+export default function ErrorBoundary({
+  children,
+}: PropsWithChildren<unknown>) {
```

Comment:
`ReactNode` is the correct type for children usually.

Reply:
I've learned to use PropsWithChildren because the type of children can change internally in React (unlikely I know) and this code would still have correct typings. Both of them are correct to my knowledge.

If you prefere `children: ReactNode`, that's ok :)

Reply:
except that when using PropsWithChildren type, the children property is optional while here for ErrorBoundary component the children are required.